Code/Framework/AzToolsFramework/AzToolsFramework/ViewportUi/ViewportUiDisplay.cpp

```diff
@@ -361,11 +374,37 @@ namespace AzToolsFramework::ViewportUi::Internal
         m_fullScreenLayout.addLayout(&m_uiOverlayLayout, 0, 0, 1, 1);
 
         // format the label which will appear on top of the highlight border
-        AZStd::string styleSheet = AZStd::string::format("background-color: %s; border: none;", HighlightBorderColor);
+        styleSheet = AZStd::string::format("background-color: %s; border: none;", HighlightBorderColor);
         m_viewportBorderText.setStyleSheet(styleSheet.c_str());
         m_viewportBorderText.setFixedHeight(ViewportUiTopBorderSize);
         m_viewportBorderText.setVisible(false);
         m_fullScreenLayout.addWidget(&m_viewportBorderText, 0, 0, Qt::AlignTop | Qt::AlignHCenter);
+
+        // format the back button which will appear in the top right of the highlight border
+        styleSheet = AZStd::string::format(
+            "border: 0px; padding-left: %dpx; padding-right: %dpx", HighlightBorderBackButtonMargin, HighlightBorderBackButtonMargin);
+        m_viewportBorderBackButton.setStyleSheet(styleSheet.c_str());
```

Comment:
We're trying to shy away from programmatically set stylesheets in Qt. I guess this works for now given the deadlines, but we may want to change it in the future.

Reply:
noted 👍